src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/JpcApplication.kt

```diff
@@ -21,11 +20,11 @@ fun main(args: Array<String>) {
       (context.getBean(ManualLocationImporter::class) as ManualLocationImporter).let { SpringApplication.exit(context, it.import()) }
     }
 
-    context.environment.getProperty("import-serco-prices")?.let {
+    context.environment.getProperty("import-SERCO-prices")?.let {
```

Comment:
We don't need any of this logic in this class now.  We can do what we need via Spring Shell.  Have I mentioned Spring Shell before? A rhetorical question.